faunadb/query.py

```diff
@@ -557,6 +557,18 @@ def not_(boolean):
   """See the `docs <https://fauna.com/documentation/queries#misc_functions>`__."""
   return _fn({"not": boolean})
 
+def to_string(expr):
+  return _fn({"to_string": expr})
```

Comment:
Do we have a section in the docs that we can link here?

Reply:
Not yet.